packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -778,4 +779,133 @@ describe('MenuTrigger', function () {
     let menu2Item1 = within(menu2).getByText('Whiskey');
     expect(menu2Item1).toBeInTheDocument();
   });
+
+  describe('MenuTrigger trigger="longPress" open behavior', function () {
+    installPointerEvent();
+
+    const DEFAULT_THRESHOLD = 500;
+
+    it('should open the menu on longPress', function () {
```

Comment:
Can we add one or more tests showing the negative case of it not opening on click, only on longPress?

Reply:
Sure, I'll work on this this week